tawazi/config.py

```diff
@@ -2,9 +2,18 @@
 
 
 class Config(BaseSettings):
+    # whether the default in tawazi is sequentiality or not.
+    # This is helpful to reduce your code size and avoid repeating @xn(is_sequentiality=True/False)
     TAWAZI_IS_SEQUENTIAL: bool = Field(False, env="TAWAZI_IS_SEQUENTIAL")
```

Comment:
if the env var has the same name as the config attribute you don't have to specify it in a Field object, pydantic will take care of that himself, you can just do :
```python
TAWAZI_IS_SEQUENTIAL: bool = False
```

Reply:
aha... will do it